tutorials/20_REST_API.ipynb

```diff
@@ -0,0 +1,538 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial: Using Haystack with REST API\n",
+    "\n",
+    "- **Level**: Advanced\n",
+    "- **Time to complete**: 30 minutes\n",
+    "- **Prerequisites**: Basic understanding of Docker, basic knowledge of Haystack pipelines \n",
+    "- **Nodes Used**: `ElasticsearchDocumentStore`, `EmbeddingRetriever`\n",
+    "- **Goal**: Learn how you can interact with Haystack through REST API."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "Haystack enables you to apply the latest NLP technology to your own data and create production-ready applications. Building an end-to-end NLP application requires the combination of multiple concepts:\n",
+    "* **DocumentStore** stores the data. You will use Elasticsearch for this tutorial.\n",
```

Comment:
If you are not familiar with Haystack this doesn't tell you much: I'd say something along the lines of 
> DocumentStore: is the component in Haystack responsible for loading and storing text data in form of Documents. In this tutorial, the DocumentStore will use Elasticsearch behind the scene